itez/beneficiary/models.py

```diff
@@ -67,28 +96,15 @@ def __str__(self):
 
 class WorkDetail(models.Model):
     """
-    Include Work Detail properties. 
+    Include Work Detail properties.
     """
-    gross_pay = models.DecimalField( 
-        _("Monthly Salary"),
-        decimal_places=2,
-        max_digits=1000,
-        null=False
-    )
-    company = models.CharField(
-        _("Company Name"),
-        max_length=200,
-        null=False
-    )
-    insured = models.BooleanField(
-        _("Company Insured"),
-        default = False
-    )
-    work_address = models.TextField(
-        _("Work Address"),
-        null=True,
-        blank=True
+
+    gross_pay = models.DecimalField(
+        _("Monthly Salary"), decimal_places=2, max_digits=1000, null=False
     )
+    company = models.CharField(_("Company Name"), max_length=200, null=False)
+    insured = models.BooleanField(_("Company Insured"), default=False)
+    work_address = models.TextField(_("Work Address"), null=True, blank=True)
```

Comment:
am wondering why apply your IDE changes when we are working against specific code formatting
change these too

Reply:
I was using black, I have disabled it for now.